nautobot_netbox_importer/tests/fixtures/netbox_dump.json

```diff
@@ -3918,7 +3918,7 @@
             "_cable_peer_type": null,
             "_cable_peer_id": null,
             "_path": null,
-            "type": "rj-45"
+            "type": "DNE"
```

Comment:
Is `DNE` being used here as a shorthand for `does not exist`? If so maybe setting it explicitly to `"does-not-exist"` might be clearer?